### PR TITLE
Content to resolve issue 14499 (add annotator to Apps)

### LINKED ISSUE #14499: Use an annotator in Lightning workflows
## 📚 Documentation

[Sagar S Karki](https://app.slack.com/team/U03E3HEHPT9)
(https://pytorch-lightning.slack.com/archives/CRBLFHY79/p1662067754013369)
I just saw a post on LinkedIn on using annotator tools in workflow.
Can some one please direct me on more resources on the.
I plan on attaching a trained model to the annotation system as a LightingWork.
Is this possible?

Yes it is possible here is an App and a Component for reference:
[tchaton](https://app.slack.com/team/U018QTT9QRL): https://github.com/Lightning-AI/lightning-pose-app
[Robert Lee](https://app.slack.com/team/U023VDYPB0Q): https://github.com/robert-s-lee/lit_label_studio

______________________________________________________________________

#### If you enjoy Lightning, check out our other projects! ⚡

- [**Metrics**](https://github.com/Lightning-AI/metrics): Machine learning metrics for distributed, scalable PyTorch applications.

- [**Lite**](https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.html): enables pure PyTorch users to scale their existing code on any kind of device while retaining full control over their own loops and optimization logic.

- [**Flash**](https://github.com/Lightning-AI/lightning-flash): The fastest way to get a Lightning baseline! A collection of tasks for fast prototyping, baselining, fine-tuning, and solving problems with deep learning.

- [**Bolts**](https://github.com/Lightning-AI/lightning-bolts): Pretrained SOTA Deep Learning models, callbacks, and more for research and production with PyTorch Lightning and PyTorch.

- [**Lightning Transformers**](https://github.com/Lightning-AI/lightning-transformers): Flexible interface for high-performance research using SOTA Transformers leveraging PyTorch Lightning, Transformers, and Hydra.

cc @Felonious-Spellfire

### PR DESCRIPTION
**Pull request by Felonious-Spellfire**
## What does this PR do?

Includes content on how to add an annotator to Apps. The App and Component referenced in this content are Early Adopter versions.

Fixes #14499

### Does your PR introduce any breaking changes? If yes, please list them.

None

## Before submitting

- [X] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [ ] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [X] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [ ] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda @rohitgr7 @Felonious-Spellfire

### CODE REVIEW

File: docs/source-app/workflows/add_annotator/label_studio/basic.rst [hunk 1]
```diff
@@ -0,0 +1,100 @@
+####################################
+Add a Label Studio annotator (basic)
+####################################
+
+**Audience:** Users who want to add a Label Studio annotator.
+
+.. warning::
+
+    The Component in this article is not in the official Lightning AI Component Gallery. It is an Early Access version of the Component.
+	The repo for this Component is here: https://github.com/robert-s-lee/lit_label_studio. Refer to the Readme for more information.
```
Comment (awaelchli) [inline docs/source-app/workflows/add_annotator/label_studio/basic.rst:10]:
```suggestion
    The Component in this article is not in the official Lightning AI Component Gallery. It is an Early Access version of the Component.
    The repo for this Component is here: https://github.com/robert-s-lee/lit_label_studio. Refer to the Readme for more information.
```

**Review by awaelchli — APPROVED** [review]
The usual comment about "python -m"
😅
